build-docker/action.yaml

```diff
@@ -96,7 +116,11 @@ runs:
           VERSION=${{ inputs.image-version }}
           REVISION=${{ inputs.image-version }}
           BRANCH=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.branch'] }}
-        push: true
+          AWS_ACCESS_KEY_ID=${{ inputs.aws-access-key-id }}
+          AWS_SECRET_ACCESS_KEY=${{ inputs.aws-secret-access-key }}
+          AWS_REGION=${{ inputs.aws-region }}
```

Comment:
Why are these files needed? Not mentioned in the PR description

Reply:
They should probably be secrets (at least the secret key)

Reply:
Updated the mechanism to dynamically input multiple build args and secrets